coverage/block-navigation.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
```

Comment:
Same as previous PR, this is auto-generated code, it doesn't serve any purpose for it to be on the repo, this is only important locally, I believe this should be part of git ignore

Reply:
Moreover this is going to be updated on every PR which would make reviewing much more complicated as there are more files to review, in this case these files are not worth reviewing as they are not created by you.